packages/modals/src/styled/StyledClose.ts

```diff
@@ -17,16 +15,46 @@ export interface IStyledCloseProps {
 }
 
 /**
- * Supports all `<button>` props.
+ * 1. Remove dotted outline from Firefox on focus.
  */
-export const StyledClose = styled.button.attrs<IStyledCloseProps>(props => ({
+export const StyledClose = styled.button.attrs<IStyledCloseProps>({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(ModalStyles['c-dialog__close'], {
-    // State
-    [ModalStyles['is-hovered']]: props.hovered
-  })
-}))<IStyledCloseProps>`
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledCloseProps>`
+  display: block;
+  position: absolute;
+  top: ${props => props.theme.space.base * 2.5}px;
+  ${props => (props.theme.rtl ? 'left' : 'right')}: ${props => `${props.theme.space.base * 5}px`};
+  transition: background-color 0.1s ease-in-out;
+  border: none;
+  border-radius: 50%;
+  background-color: transparent;
+  cursor: pointer;
+  padding: 0;
+  width: ${props => props.theme.space.base * 10}px;
+  height: ${props => props.theme.space.base * 10}px;
+  overflow: hidden;
+  text-decoration: none;
+  color: ${getColor('neutralHue', 600)};
```

Comment:
Does this need a `props.theme` argument?

Reply:
It seems like the `getColor` call here returns the correct value: `#68737d`, so it does work in this PR. Do you know if there are cases where not passing in the third argument (`theme`) results in unintended values?

I'll go with the safer approach here, and use `${props => getColor('neutralHue', 600, props.theme)}`. Thanks for pointing this out!